### PR TITLE
Modals validation and UI improvements

### PR DESCRIPTION
**Pull request by MariaPadalka**
## Description:
Fixed an issue with key fields and improved the UI for creating social links. A lot of changes, because I used _Prettier_ to make the code look better.

## DONE CHECK LIST
- [x] all data is inserted into the database
- [x] no bad requests
- [x] all required fields are taken into account

### CODE REVIEW

File: src/features/AdminPage/PartnersPage/PartnerModal/PartnerModal.component.tsx [hunk 1]
```diff
@@ -1,367 +1,495 @@
...
+    const onSuccesfulSubmitPartner = async (formValues: any) => {
+      if (showSecondForm) {
+        try {
+          await partnerLinksForm.validateFields();
+          console.log(partnerLinksForm.getFieldsValue());
+          onSuccesfulSubmitLinks(partnerLinksForm.getFieldsValue());
+          console.log(5);
+        } catch (errorInfo) {
+          message.error("Будь ласка, заповніть всі обов'язкові поля");
+          return; // Exit the function here
```
Comment (MaksBrat) [inline src/features/AdminPage/PartnersPage/PartnerModal/PartnerModal.component.tsx:164]: do you need this comment?

**Review by MaksBrat — CHANGES_REQUESTED** [review]

File: src/features/AdminPage/PartnersPage/PartnerModal/PartnerModal.component.tsx [hunk 2]
```diff
@@ -1,367 +1,495 @@
...
-         form.resetFields();
-         partnerLinksForm.resetFields();
-         selectedStreetcodes.current = [];
-         partnerSourceLinks.splice(0);
-         setIsModalOpen(false);
-     };
+    const onSuccesfulSubmitLinks = (formValues: any) => {
+      const url = formValues.url as string;
+      const logotype = partnerLinksForm.getFieldValue("logotype");
+      console.log(url, logotype);
```
[the remark this answers is not in the document]
Comment (MariaPadalka) [reply src/features/AdminPage/PartnersPage/PartnerModal/PartnerModal.component.tsx]: resolved)

File: src/features/AdminPage/PartnersPage/PartnerModal/PartnerModal.component.tsx [hunk 3]
```diff
@@ -1,367 +1,495 @@
...
+      setShowSecondForm(false);
+      setShowSecondFormButton(true);
+    };
+    const onSuccesfulSubmitPartner = async (formValues: any) => {
+      if (showSecondForm) {
+        try {
+          await partnerLinksForm.validateFields();
+          console.log(partnerLinksForm.getFieldsValue());
+          onSuccesfulSubmitLinks(partnerLinksForm.getFieldsValue());
+          console.log(5);
```
[the remark this answers is not in the document]
Comment (MariaPadalka) [reply src/features/AdminPage/PartnersPage/PartnerModal/PartnerModal.component.tsx]: resolved)

File: src/features/AdminPage/PartnersPage/PartnerModal/PartnerModal.component.tsx (hunk 1 above)
Comment (MariaPadalka) [reply src/features/AdminPage/PartnersPage/PartnerModal/PartnerModal.component.tsx]: deleted it

**Review by MaksBrat — APPROVED** [review]

**Review by KateYatsiuk — APPROVED** [review]

**Review by ormykhalyshyn — APPROVED** [review]

**Review by Tysyatsky — APPROVED** [review]

File: src/features/AdminPage/PartnersPage/PartnerModal/PartnerModal.styles.scss [hunk 4]
```diff
@@ -24,19 +24,69 @@
...
-            $wrap: nowrap,
-            $gap: 5px)
-    }
+.save{
+    min-height: 50px;
+}
+.link-container{
+    padding:5px;
+    margin-bottom: 20px;
+    border: solid rgb(237, 231, 231) 1px ;
```
Comment (OlehHatsko) [inline src/features/AdminPage/PartnersPage/PartnerModal/PartnerModal.styles.scss:49]: Color variables?

**Pull request merged**